app/controllers/search.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const User = mongoose.model('User');
+
+/**
+ * Gets all users from the database
+ */
+exports.users = (req, res) => {
+  const query = req.params.inviteeEmail || '';
+  User.find({ email: { $regex: query } }).limit(10)
```

Comment:
When typing in an email address, a list of email suggestions that partly match what is typed is displayed. The limit() is to limit the number of email suggestions that are displayed for you to pick from.